packages/sitecore-jss-react/src/components/FEaaSComponent.tsx

```diff
@@ -1,94 +1,131 @@
-/* eslint-disable react/prop-types */
-import React, { useEffect } from 'react';
-import { ComponentParams, ComponentFields, getFieldValue } from '@sitecore-jss/sitecore-jss/layout';
-// eslint-disable-next-line
-import type * as _FEAAS from '@sitecore-feaas/clientside';
+import React from 'react';
+import * as FEAAS from '@sitecore-feaas/clientside/react';
+import { ComponentFields, getFieldValue } from '@sitecore-jss/sitecore-jss/layout';
 
 export const FEAAS_COMPONENT_RENDERING_NAME = 'FEaaSComponent';
-export const FEAAS_MODULE_SRC = 'https://feaasstatic.blob.core.windows.net/packages/clientside/latest/browser/index.esm.js';
 
-export type FEaaSComponentParams = ComponentParams & {
+export type FEaaSComponentParams = {
   LibraryId?: string;
   ComponentId?: string;
   ComponentVersion?: string;
-  ComponentRevision?: string;
+  ComponentRevision?: number | 'staged' | 'published' | 'saved';
   ComponentHostName?: string;
   ComponentInstanceId?: string;
   ComponentDataOverride?: string;
   ComponentHTMLOverride?: string;
+  styles?: string;
+  RenderingIdentifier?: string;
 };
 
-export type FEaaSComponentProps = {
+/**
+ * FEaaS props for server rendering.
+ */
+type FEaaSComponentServerProps = {
+  template?: string;
+  lastModified?: string;
+}
+
+/**
+ * FEaaS props for client side component. Should be used as fallback when server props are not provided.
+ */
+type FEaaSComponentClientProps = {
+  src?: string;
```

Comment:
Should `src` be moved to `FEaaSComponentServerProps`? This would match our usage of it (in the wrapper's `getServerSide/StaticProps` to enable server rendering)

Reply:
Having talked with Yarik some more, I removed src altogether to reduce complexity and the amount of places things could break at.